### PR TITLE
Bump minSDK to 21 and remove mistakenly submitted files adding them t…

### PR DESCRIPTION
**Pull request by artakka**
…o .gitignore

### CODE REVIEW

File: platform/android/MapboxGLAndroidSDKLint/.gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 /build
+local.properties
+
```
Comment (wipfli) [inline platform/android/MapboxGLAndroidSDKLint/.gitignore:3]: Did you delete the file in a different branch?

File: platform/android/gradle/dependencies.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ext {
 
     androidVersions = [
-            minSdkVersion    : 14,
+            minSdkVersion    : 21,
```
Comment (wipfli) [inline platform/android/gradle/dependencies.gradle:4]: I would add a changelog item for this version bump.

File: platform/android/MapboxGLAndroidSDKLint/.gitignore (hunk 1 above)
Comment (artakka) [reply platform/android/MapboxGLAndroidSDKLint/.gitignore:3]: Not intentionally.

File: platform/android/gradle/dependencies.gradle (hunk 2 above)
Comment (wipfli) [reply platform/android/gradle/dependencies.gradle:4]: You can edit platform/android/changelog.md

File: platform/android/MapboxGLAndroidSDKLint/.gitignore (hunk 1 above)
Comment (wipfli) [reply platform/android/MapboxGLAndroidSDKLint/.gitignore:3]: Ah OK then probably you can remove the file with this pull request.

Comment (wipfli) [reply platform/android/MapboxGLAndroidSDKLint/.gitignore:3]: https://github.com/maplibre/maplibre-gl-native/blob/main/platform/android/MapboxGLAndroidSDKLint/local.properties is still on main...

**Comment by wipfli** [comment]
Sorry for nagging but can you add a link to the pull request in the changelog? See examples in other changelog entries.

**Review by wipfli — APPROVED** [review]
Please add the changelog link, then this is good from my side. Thanks @artakka

**Comment by artakka** [comment]
@wipfli  No problem. Good to be consistent.

**Pull request merged**